src/modules/adaptors/Morpho/MorphoBlue/MorphoBlueHelperLogic.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { IMorpho, MarketParams, Market, Id } from "src/interfaces/external/Morpho/MorphoBlue/interfaces/IMorpho.sol";
+import { MathLib, WAD } from "src/interfaces/external/Morpho/MorphoBlue/libraries/MathLib.sol";
+import { SharesMathLib } from "src/interfaces/external/Morpho/MorphoBlue/libraries/SharesMathLib.sol";
+import { IOracle } from "src/interfaces/external/Morpho/MorphoBlue/interfaces/IOracle.sol";
+import { UtilsLib } from "src/interfaces/external/Morpho/MorphoBlue/libraries/UtilsLib.sol";
+import { MorphoLib } from "src/interfaces/external/Morpho/MorphoBlue/libraries/periphery/MorphoLib.sol";
+import { console } from "@forge-std/Test.sol";
+
+/**
+ * @title Morpho Blue Helper contract.
+ * @notice Helper implementation including health factor logic used by Morpho Blue Adaptors.
+ * @author 0xEinCodes, crispymangoes
+ * NOTE: helper functions made virtual in case future Morpho Blue Pair versions require different implementation logic.
+ */
+contract MorphoBlueHelperLogic {
+    // Using libraries from Morpho Blue codebase to ensure same mathematic methods
+    using MathLib for uint128;
+    using MathLib for uint256;
+    using UtilsLib for uint256;
+    using SharesMathLib for uint256;
+    using MorphoLib for IMorpho;
+
+    /**
+     * @notice The Morpho Blue contract on current network.
+     */
+    IMorpho public immutable morphoBlue;
+
+    // Constant from Morpho Blue
+    uint256 constant ORACLE_PRICE_SCALE = 1e36;
```

Comment:
This constant is available in `ConstantsLib`

Reply:
True, but I've removed the `ConstantsLib` now since we would only use the `ORACLE_PRICE_SCALE` constant from it.